### DISCUSSION TITLE
docker安装的nanomq:0.8.0-slim如何设置配置项

### DISCUSSION THREAD

**Discussion by zhuyujie1993** [nanomq/nanomq#881] [Q&A] [answered]
主要是消息桥接部分 重点是sqlite落盘、断网续传的配置

**Comment by JaylinYu** [comment]
容器启动的话 建议通过环境变量配置  https://github.com/emqx/nanomq#nanomq-environment-variables
现在桥接消息通过sqlite落盘和断网自动续传是需要另外开启SQLite选项编译生效的。从dockerhub下载的是没有开启此项功能的。

**Comment by JaylinYu** [comment]
这个月内会推出默认开启sqlite的容器镜像。

**Comment by zhuyujie1993** [comment]
> 容器启动的话 建议通过环境变量配置 https://github.com/emqx/nanomq#nanomq-environment-variables 现在桥接消息通过sqlite落盘和断网自动续传是需要另外开启SQLite选项编译生效的。从dockerhub下载的是没有开启此项功能的。
现在落盘的化只能额外编译sqlite达到，现在的版本有没有一些落盘和自动续传的环境配置的

**Comment by zhuyujie1993** [comment]
> 这个月内会推出默认开启sqlite的容器镜像。

就是在在容器中直接内嵌的sqlite的选项，可以通过conf文件配置么，就像emqx的类似

**Comment by zhuyujie1993** [comment]
> > 这个月内会推出默认开启sqlite的容器镜像。
> 
> 就是在在容器中直接内嵌的sqlite的选项，可以通过conf文件配置么？

**Comment by zhuyujie1993** [comment]
还有一个疑问 大概在哪个阶段有nanomq的可视化套件？ 毕竟在边缘设备上需要运维人员灵活配置

**Comment by JaylinYu** [comment, marked as answer]
> > 这个月内会推出默认开启sqlite的容器镜像。
> 
> 就是在在容器中直接内嵌的sqlite的选项，可以通过conf文件配置么，就像emqx的类似

目前的容器只是最小功能子集，额外功能都需要编译打开。 不过如果你需要的话可以制作安装包来完成。sqlite内嵌也还不支持通过配置打开，请另外开一个issue我们后面支持一下。

可视化套件后续会做成云配置类型，远程通过隧道 + HTTP API来进行管理。
